test/micro/org/openjdk/bench/jdk/incubator/foreign/LoopOverNonConstantMapped.java

```diff
@@ -147,15 +146,15 @@ public int segment_loop() {
     public int segment_loop_static() {
         int res = 0;
         for (int i = 0; i < ELEM_SIZE; i ++) {
-            res += MemoryAccess.getIntAtIndex(segment.address(), i);
+            res += MemoryAccess.getIntAtIndex(segment, i * CARRIER_SIZE);
```

Comment:
Why this multiply by CARRIER_SIZE? That should be done by the accessor.

Reply:
mistake - will fix (but this has nothing to do with this patch, I think?)

Reply:
Well, this patch adds the multiply...

Reply:
> Well, this patch adds the multiply...

Ugh - that's a bug then